cmake_templates/qgsconfig.h.in

```diff
@@ -98,7 +98,7 @@
 
 #cmakedefine HAVE_EPT
 
-#cmakedefine HAVE_PDAL
+#cmakedefine HAVE_PDAL_QGIS
```

Comment:
Slightly off topic but ...
Should we "generically" change this, e.g. the lines below (versioning) are likely to be set by pdal headers itself one day and we'll have conflicts as soon as that happens. I.e. this would be a similar concept like the `Qgs` prefix for classes.

Reply:
Yes, I think that would be a good idea...